lib/index.js

```diff
@@ -165,6 +165,7 @@ const exec = async (opts) => {
           const prompt = `Need to install the following packages:\n${
           addList
         }Ok to proceed? `
+          log.clearProgress()
```

Comment:
You'll need to do a check here to make sure this method exists, since `log` is [proc-log](https://github.com/npm/proc-log) by default, which doesn't have that method.

Reply:
Thank you! Updated